v2/nodejs/jwt/init.mdx

```diff
@@ -0,0 +1,25 @@
+---
+id: init
+title: init
+hide_title: true
+---
+
+```jsx
+JWTRecipe.init({
```

Comment:
Why is this called JWTRecipe here? No other recipe has the word Recipe postfix to it.

Reply:
Ive been using JWTRecipe in the NodeJS tests, just defaulted to that here. Changed it to `JWT`